VideoLocker/src/main/java/org/edx/mobile/player/PlayerFragment.java

```diff
@@ -259,6 +259,22 @@ protected void onUpdate() {
                     isLastRotationOn = isRotationOn;
                 }
             };
+            getView().findViewById(R.id.panel_video_only_on_web).setOnClickListener(new OnClickListener() {
+                @Override
+                public void onClick(View v) {
+
+                    final StringBuffer urlStringBuffer = new StringBuffer();
+                        if (! videoEntry.url.startsWith("http://") && ! videoEntry.url.startsWith("https://")) {
```

Comment:
This probably needs a comment. Do we have URLs (besides the invalid one) that don't start with ``http`` or ``https``? And if it is the invalid one, we probably don't want to open it?